compiler/src/main/scala/org/scalajs/core/compiler/PrepJSInterop.scala

```diff
@@ -211,6 +211,20 @@ abstract class PrepJSInterop extends plugins.PluginComponent
 
         enterOwner(OwnerKind.NonEnumScalaMod) { super.transform(modDef) }
 
+      /* Identify and fix accessor that have the shape described in issue #1899.
+       *
+       * This fix is uses an over approximation of the shape of the tree due to
+       * loss of information from bug SI-9487. As such the fix introduces a
+       * difference in semantics for new js.Object {...}. Any private val,
```

Comment:
"a difference in semantics" is a big too gentle. I would write "introduces another bug"